Lib/ntpath.py

```diff
@@ -272,6 +272,8 @@ def ismount(path):
         return True
 
     if _getvolumepathname:
+        if not lexists(path):
```

Comment:
What if the path gets removed right after this line executes?

Reply:
Oops! So I almost introduced a potential bug for fixing a bug...

How about explicitly catching the `FileNotFoundError` that could be thrown by `_getvolumepathname`?

```suggestion
        try:
            return path.rstrip(seps) == _getvolumepathname(path).rstrip(seps)
        except FileNotFoundError:
            return False
```

Reply:
Hi Chris @lazka

Could you please review my update?

Reply:
lgtm